src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/PopulateUpdateSubmission.java

```diff
@@ -151,5 +153,5 @@ private void populateFilingForDate(
                             .ifPresent(updateSubmission::setFilingForDate);
                 });
     }
-
+    Function <Integer, String> toStringDate = value -> String.format("%02d",value);
```

Comment:
As an instance variable, this should probably be at the top of the class 

That said as it's only used in this method I think it should be fine to just define it in `populateFilingForDate()` or even just put the lambda expression in the `map` itself

Reply:
ooh yeah, gonna insert the lambda exp straight in, thanks!